styles.css

```diff
@@ -0,0 +1,389 @@
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.navbar {
+  background-color: white;
+}
+
+.navbar-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 40px;
+  margin-right: 40px;
+  margin-top: 40px;
+  border: none;
+}
+
+.navbar-content img {
+  width: 20px;
+  height: 13px;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.nav_link ul {
+  display: none;
+}
+
+.section-1 {
+  height: 720px;
+  background-color: #f7f7f9;
+  border-style: none;
+}
+
+.section-1-container {
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-bottom-left-radius: 100px;
+  background-color: white;
+  padding: 160px 25px;
+  width: 100%;
+  height: 100%;
+}
+
+.heading {
+  font-size: 2.5em;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+.leading {
+  font-size: 18px;
+  text-align: start;
+  margin: 20px 0;
+}
+
+.social-heading {
+  color: #6070ff;
+  margin-bottom: 8px;
+}
+
+.socials {
+  display: flex;
+  list-style: none;
+}
+
+.socials img {
+  padding-right: 15px;
+}
+
+.section-2 {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  justify-content: center;
+  background-color: #f7f7f9;
+  padding-bottom: 100px;
+}
+
+.container-1 img {
+  padding: 20px 20px;
+  width: 100%;
+}
+
+.container-1 {
+  background-color: white;
+  margin-left: 1.2em;
+  margin-right: 1.2em;
+  justify-self: center;
+  margin-top: 100px;
+  border-radius: 20px;
+  border: 1px solid #dfe1e6;
+  padding-bottom: 20px;
+}
+
+.items ul {
+  display: flex;
+  font-size: 12px;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.items {
+  padding-left: 20px;
+  margin-right: 50px;
+}
+
+.items h1 {
+  font-size: 2em;
+}
+
+.items p {
+  font-size: 16px;
+  text-align: start;
+  margin-right: 25px;
+  margin-bottom: 10px;
+}
+
+.items ul li {
+  padding-right: 5px;
+  font-weight: 700;
+  font-size: 1.2em;
+  margin-bottom: 10px;
+}
+
+.items ul li:nth-child(1) {
+  list-style: none;
+  margin-right: 10px;
+  font-weight: 500;
+}
+
+.items ul li:nth-child(2) {
+  color: #7a869a;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-weight: 400;
+}
+
+.items ul li:nth-child(3) {
+  color: #7a869a;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-weight: 400;
+}
+
+.btn {
+  border: none;
+  padding: 4px 12px;
+  color: #6070ff;
+  text-decoration: none;
+  background-color: #ebebff;
+  border-radius: 10px;
+  margin-right: -20px;
+  font-weight: 500;
+}
+
+.btn-div {
+  margin: 15px 0;
+  list-style: none;
+  font-size: 12px;
+  display: flex;
+  flex-direction: row;
+}
+
+.last-btn {
+  border: 1px solid #6070ff;
+  padding: 10px 18px;
+  border-radius: 10px;
+  margin-top: 8px;
+  background-color: white;
+  color: #6070ff;
+  font-size: 20px;
+  font-weight: 600;
+}
```

Comment:
- [ ] Also we have to implement the button interactions for all the buttons on our page, and the font-size for these butoons should be 17px and font-weight 500 according to figma please correct that currently it is 20 px and 600 font-weight
![image](https://user-images.githubusercontent.com/97521557/235291932-0b707935-fd24-4fce-85e4-54e5af7fbe3f.png)
 
here is the button interactions design from figma :point_down: 
![image](https://user-images.githubusercontent.com/97521557/235291969-fab4bb24-ebf6-4ebd-8ba3-2bde80795f74.png)

please do this for all the buttons on your page